app/views/documents/index/_filters.html.erb

```diff
@@ -6,7 +6,12 @@
     },
     name: "title_or_url",
     value: params[:title_or_url],
-    type: "search"
+    type: "search",
+    data: {
+      gtm: "input-title-or-url",
+      "gtm-visibility-tracking": true,
+      "gtm-action": params[:title_or_url],
```

Comment:
Will this and the others fire for blank attributes? if so do we want that?

Reply:
Indeed it will. We do this at the moment and I'm relying on that to make this generic. I think it's OK to keep firing as it helps us understand, in a relative way how, often each of the filters is actually used; if we didn't want to see blank events, we can just filter them out later on. If we want to avoid blank events, we could do that with a special tag, but perhaps that can be a future iteration?